wrappers/ios/libindy-pod/Indy-demoTests/Test Utils/AnoncredsUtils.m

```diff
@@ -17,17 +17,30 @@ @implementation AnoncredsUtils
 
 + (AnoncredsUtils *)sharedInstance {
     static AnoncredsUtils *instance = nil;
-    static dispatch_once_t dispatch_once_block;
 
-    dispatch_once(&dispatch_once_block, ^{
-        instance = [AnoncredsUtils new];
-        instance.isCommonWalletCreated = false;
-        instance.walletHandle = 0;
-    });
+    @synchronized(self) {
+        if (instance == nil) {
+            instance = [AnoncredsUtils new];
+            instance.isCommonWalletCreated = false;
+            instance.walletHandle = 0;
+        }
+    }
 
     return instance;
 }
 
++ (void)clearInstance {
```

Comment:
This is probably not the best approach but after credentials are removed wallet has to be recreated in order for other tests to work as tests use singleton instance. I'm open to better suggestions.